Providers/nxAvailableUpdates/MSFT_nxAvailableUpdatesResource.cpp

```diff
@@ -228,6 +228,10 @@ void MI_CALL MSFT_nxAvailableUpdatesResource_Invoke_InventoryTargetResource(
 		FILE *fp = NULL;
 		
 		clientBuffer = (MI_Uint8*)malloc(clientBufferLength + 1);
+        if(clientBuffer == NULL) {
+            return;

```

Comment:
We do not own this resource,. We should work with UM team.

Reply:
Reverted. Some formatting(spacing) changes are shown in the file which can be ignored.